CHANGELOG.rst

```diff
@@ -7,6 +7,9 @@ CHANGELOG
 
 * doc-fix: update information about saving models in the MXNet README
 * doc-fix: change ReadTheDocs links from latest to stable
+* feature: Support for predictor to delete endpoint, and delete endpoint configuration with ``delete_endpoint()`` by default
```

Comment:
I think the ability to delete endpoint has always been there. This should reflect that the ability to delete endpoint configuration was added.